index.html

```diff
@@ -69,34 +73,54 @@ <h2 class="fw-bold">2021.12.17(FRI) - 19(SUN)</h2>
             <h2 class="text-center">Main Program</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr class="text-center">
 
             <div class="container d-flex flex-row flex-wrap">
                 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
+                    <img class="col-2 img-cards" src="./assets/work.svg" alt="">
+                    <h2 class="text-primary text-primary-card  col-4">Intro</h2>
+                    <p class="text-light text-p-card col-6">
+                        <small>
+                            some text must to be inserted here
+                        </small>
+                    </p>
+                </div>
+
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
+                    <img class="col-2 img-cards" src="./assets/work.svg" alt="">
+                    <h2 class="text-primary text-primary-card  col-4">Contest</h2>
+                    <p class="text-light text-p-card col-6">
+                        <small>
+                            some text must to be inserted here
+                        </small>
+                    </p>
+                </div>
+
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
-                    <h2 class="text-primary  col-4">Intro</h2>
-                    <p class="text-light col-6">
+                    <h2 class="text-primary text-primary-card  col-4">Prizing</h2>
+                    <p class="text-light text-p-card col-6">
                         <small>
                             some text must to be inserted here
                         </small>
                     </p>
                 </div>
 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
-                    <h2 class="text-primary  col-4">Contest</h2>
-                    <p class="text-light col-6">
+                    <h2 class="text-primary text-primary-card  col-4">Prizing</h2>
+                    <p class="text-light text-p-card col-6">
                         <small>
                             some text must to be inserted here
                         </small>
                     </p>
                 </div>
 
-                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400">
+                <div class="container-sm bg-dark-hopacity row mb-1 ms-1 pt-2 pb-2 tw-400-card">
                     <img class="col-2 img-cards" src="./assets/work.svg" alt="">
```

Comment:
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐